src/Compilers/CSharp/Portable/Symbols/Synthesized/RefKindVector.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal struct RefKindVector : IEquatable<RefKindVector>
+    {
+        private BitVector _bits;
+
+        internal static RefKindVector Create(int capacity)
+        {
+            return new RefKindVector(capacity);
+        }
+
+        private RefKindVector(int capacity)
+        {
+            _bits = BitVector.Create(capacity * 2);
+        }
+
+        internal bool IsNull => _bits.IsNull;
+
+        internal int Capacity => _bits.Capacity / 2;
+
+        internal IEnumerable<ulong> Words() => _bits.Words();
```

Comment:
I'm surprised dynamic tests weren't affected by the change from `BitVector` to `RefKindVector`. `MakeDynamicCallSiteDelegateName` uses those "words" as part of the name for the synthesized delegate type.